### PR TITLE
Music staff canvas

### PR DESCRIPTION
**Pull request by ArielSalazar04**
In this pull request, I made a canvas that resembles a music staff and made it capable of receiving the user's drawings. I also added a "clear" button in case the user wants to change a measure or erase all of his or her content. 

beats35

### CODE REVIEW

**Review by desireepenaranda — APPROVED** [review]
Ariel the grid is fantastic. And the drawing of the shapes came out so cool!

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -11,5 +11,17 @@
       <h1>Welcome to Beats 35!</h1>
       <p> Not just music but shapes too!</p>
     </div>
+    <img src="images/clef.png" style="float: left;" width="290" height="500"></img>
```
Comment (jetafese) [inline src/main/webapp/index.html]: Would it be possible to add the styles into our css file?

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -10,4 +10,90 @@
...
+    }, false);
+
+    var onPaint = function() {
+        ctx.lineTo(mouse.x, mouse.y);
+        ctx.stroke();
+    };
+}
+
+function clearAll(){
+	const canvas = document.getElementById('paint');
```
Comment (jetafese) [inline src/main/webapp/script.js:86]: I'm curious as to the purpose of this indentation. Does it do something special?

**Review by jetafese — APPROVED** [review]
Cool to see this develop. Left some comment to help categorize the css :)

**Pull request merged**